src/pfe/portal/modules/Project.js

```diff
@@ -129,6 +130,8 @@ module.exports = class Project {
     this.links = new Links(this.projectPath(), args.links);
 
     this.perfDashboardPath = `/performance/charts?project=${this.projectID}`;
+    log.debug(`Creating LoadRunner for user ${this.user_id}`);
```

Comment:
`Project.js` has no field called user.id. (I think you probably copied this message from `User.js`.) Should you output the projectID/name here?